CMakeLists.txt

```diff
@@ -460,7 +463,7 @@ if(WITH_MGR)
 endif()
 
 # require minimally the bundled version
-find_package(Boost 1.61 COMPONENTS thread system regex random program_options date_time iostreams REQUIRED)
+find_package(Boost 1.61 COMPONENTS ${BOOST_COMPONENTS} REQUIRED)
```

Comment:
Yes, I introduced BOOST_COMPONENTS for that purpose, but lost track of it debugging builds.  If it works correctly, it's slightly cleaner.